automatic/prtgdesktop/prtgdesktop.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>prtgdesktop</id>
+    <title>PRTG Desktop</title>
+    <version>20.7.0</version>
+    <authors>Paessler AG</authors>
+    <owners>BBT Software AG</owners>
+    <projectUrl>https://www.de.paessler.com/prtg-desktop-app</projectUrl>
```

Comment:
Don't link to the German site. Use either a generic link which detects language or if this does not exist link to English site.